Source/JavaScriptCore/wasm/WasmAirIRGenerator64.cpp

```diff
@@ -274,13 +274,40 @@ class AirIRGenerator64 : public AirIRGeneratorBase<AirIRGenerator64, TypedTmp> {
         result = tmpForType(Types::V128);
 
         if (isX86()) {
+            if (airOp == B3::Air::VectorPopcnt) {
+                ASSERT(info.lane == SIMDLane::i8x16);
+
+                // x86_64 does not natively support vector lanewise popcount, so we emulate it using multiple
+                // masks.
+
+                v128_t bottomNybbleConst, popcntConst;
```

Comment:
nit

Reply:
Split them to separate lines as well in the latest commit.